tests/experimental/wgit/cli/test_cli.py

```diff
@@ -3,11 +3,64 @@
 # This source code is licensed under the BSD license found in the
 # LICENSE file in the root directory of this source tree.
 
+
+import pathlib
+import shutil
+
 import experimental.wgit.cli as cli
 
 
+def setup_module(module):
+    try:
+        shutil.rmtree(".wgit")
```

Comment:
can we run the test inside a dir like `wgit_test_dir`, instead of the top level fairscale dir?

Reply:
Could you clarify this a little bit?

Reply:
I assume that you run the test within the current dir during development. That means, say your fariscale is at

/home/name/fairscale

then, the test will create

/home/name/fairscale/.wgit

I am wondering can you make the test run within a dir like /home/name/fairscale/wgit_unittest or something like that.

Am I missing something?

Reply:
Oh got it. Yeah, we can definitely do that.